blackbox/docs/sql/scalar.txt

```diff
@@ -1206,7 +1206,7 @@ The ``array_unique`` function merges two arrays into one array with unique eleme
 
 ::
 
-    cr> select array_unique([1,2,3],[3,4,5,6]) from sys.cluster;
+    cr> select array_unique([1, 2, 3], [3, 4, 5, 6]) from sys.cluster;
```

Comment:
mmm I mean inside the subarray: like [3, 4, 5, 3, 6, 4] or am I missing something?